src/components/trashlookup.jsx

```diff
@@ -1,156 +1,157 @@
-import React, { Component } from "react";
+import React from "react";
+import Form from "./common/Form";
 import PostData from "../Data/street.json";
-import SearchBox from "./searchBox";
+import { Button } from 'reactstrap';
 import _ from "lodash";
-import ListGroup from "./listGroup";
-import Trash from "./common/trash";
-import Leaf from "./common/leaf";
-import Recycle from "./common/recycle";
+
+
 import { getTrashService } from "../services/trashService";
+import * as moment from "moment";
+import ScheduleTable from './common/scheduleTable';
+class TrashLookUp extends Form {
+ 
+  displayText() {
+    const selectedAddress = _.trim(this.state.value);
+    const linkMessage = "Not the right address?";
+      let message = [];
+      message.push(<h6>Your Schedule</h6>);
+      message.push("showing collection schedule for:");
+      message.push(<h6> {selectedAddress} </h6>);
+      message.push("Not the right address?Try another search");
+   
+
+      const displayMessage = message.map((data, index) => {
+        return <span key={index}>{data}</span>;
+      });
+
+    return(
+     
+<div>
+   {displayMessage} <Button color="link" onClick={this.handleClick}>Try another search</Button>
+   </div>               
+    );
+  }
+
+  getAddresses() {
+    let searchQuery =  _.trim(this.state.value);
+    let filtered =PostData;
+    const isHidden = this.state.isHidden;
+ 
+    if (searchQuery.length > 0) {
+   
+         if (isHidden) 
+         {
+          filtered = PostData.filter(m =>
+            m.address1.toLowerCase().indexOf(searchQuery.toLowerCase()) > -1);
+          }
+          else{
+            filtered = PostData.filter(
+             m => m.address1.toLowerCase() === searchQuery.toLowerCase());
+      
+          }
+        }
+    return {  data: filtered };
+  }
+
+   renderConditionTable() {
```

Comment:
i would refactor this out to it's own component maybe named something like `ScheduleMessage` or `ScheduleLink`

Reply:
this one . I do not understand what you want me to do

Reply:
@sgrg1 when you pushed your changes, it looks like the code moved. I will look tomorrow and find what code this was actually pointing to.